engine/src/test/scala/cromwell/webservice/MetadataBuilderActorSpec.scala

```diff
@@ -43,7 +43,7 @@ class MetadataBuilderActorSpec extends TestKitSuite with AsyncFlatSpecLike with
 
     val mba = system.actorOf(
       props = MetadataBuilderActor.props(readMetadataWorkerMaker, 1000000),
-      name = "mba",
+      name = s"mba-${queryReply.workflowId}",
```

Comment:
This is an unrelated change, but this test kept falling due to the non-unique name of the actor.